concepts/05 Widgets/TreeList/20 Editing/10 User Interaction/20 Cell Mode.md

```diff
@@ -46,6 +46,53 @@ Clicking a *"Delete"* button invokes the confirmation dialog that allows a user
         ],
         // ...
     })
+
+##### Vue
```

Comment:
Angular and jQuery examples here still use the old approach to hiding the message. Please double check all the topics.